### PR TITLE
Update covid-19 page with postponing details

### PR DESCRIPTION
**Pull request by antoinepetty**
Modelled from teams email

### CODE REVIEW

File: covid-19.md [hunk 1]
```diff
@@ -10,37 +10,19 @@ Last updated: {{ page.date | date: "%e %B %Y" }}.
 
 ## The Competition
 
-**We currently intend to have the competition on the 18th and 19th April 2020 as planned**, however are actively and carefully considering whether this needs to change.
+Following guidance from the UK government and the University of Reading, **we will need to postpone Student Robotics 2020** from the original date in April, until some time late June / early July.
 
-Considering this is an event for minors, we will be taking decisions with an abundance of caution.
+We have sent a poll to team leaders and volunteers to allow us to select the most appropriate date.
```
Comment (Adimote) [inline covid-19.md]:
```suggestion
We have requested feedback from team leaders and volunteers to help select the most appropriate date.
```

File: _posts/2020-03-16-sr2020-competition-postponed.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- 27th - 28th June
+- 4th - 5th July
+- 11th - 12th July
+
+The event will still take place over a weekend, at the University of Reading.
+
+We have requested feedback from team leaders and volunteers to help select the most appropriate date.
+
+We hope to confirm the updated date in the next week, so please let us know as soon as you can.
```
Comment (antoinepetty) [inline _posts/2020-03-16-sr2020-competition-postponed.md]: On second thoughts - is this better to be kept in just the email?

Comment (RealOrangeOne) [reply _posts/2020-03-16-sr2020-competition-postponed.md]: Ooh, good catch!

```suggestion
We hope to confirm the updated date in the next week, so watch this space!
```

File: _events/sr2020/competition.md [hunk 3]
```diff
@@ -14,9 +14,7 @@ Keep an eye on this page for updates as more information becomes available.
 
 ## Schedule
 
-The competition will be held from *Saturday 18th April* until *Sunday 19th April*.
```
Comment (PeterJCLaw) [inline _events/sr2020/competition.md]: Maybe replace this with a comment about roughly when we hope to have a confirmed date and/or just saying that the date is yet to be confirmed.

File: covid-19.md [hunk 4]
```diff
@@ -10,37 +10,19 @@ Last updated: {{ page.date | date: "%e %B %Y" }}.
...
-### In the case of the competition being postponed
-
-We are considering weekends in late June / early July as alternatives for the competition. We will be surveying team leaders to ensure we select the most appropriate date.
-
 ## Tech Days
 
-There are no current planned tech days before the current competition date in April.
-
-If the competition is postponed, we will try and schedule some additional ones closer to the new event date.
+There are no current planned tech days before the current competition date in April. Once things have settled down, we hope to organise some.
```
Comment (PeterJCLaw) [inline covid-19.md]: The planned date is no longer April.

**Comment by PeterJCLaw** [comment]
Thanks for putting this together so quickly, just some small suggestions :+1:

File: _posts/2020-03-16-sr2020-competition-postponed.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+layout: news
+title: Student Robotics 2020 Postponed
```
Comment (RealOrangeOne) [inline _posts/2020-03-16-sr2020-competition-postponed.md]:
```suggestion
title: Student Robotics 2020 Competition Postponed
```

File: _posts/2020-03-16-sr2020-competition-postponed.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+Following guidance from the UK government and the University of Reading, **we will need to postpone Student Robotics 2020** from the original date in April, until some time late June / early July.
```
Comment (RealOrangeOne) [inline _posts/2020-03-16-sr2020-competition-postponed.md]:
```suggestion
Following guidance from the UK government and the University of Reading, **we will need to postpone the Student Robotics 2020 Competition** from the original date in April, until some time late June / early July.
```

**Pull request merged**